### PR TITLE
fix: 357-Generic-ScrollBar

### LINKED ISSUE #357: BUGS: Implement a generic Scrollbar that we can use everywhere
Image

### PR DESCRIPTION
**Pull request by MNSM92**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #357

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/ui/components/elements/GenericScrollbar/GenericScrollbar.styled.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+export const Scrollbar = styled.div<ScrollbarProps>`
+  float: left;
+  height: ${(props) => (props?.height ? props?.height : '200px')};
+  width: ${(props) => (props?.width ? props?.width : 'auto')};
+  background: ${(props) => (props?.color ? props?.color : '#F5F5F5')};
+  overflow-y: scroll;
+
+  &::-webkit-scrollbar-track {
+    -webkit-box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.3);
+    border-radius: 10px;
```
Comment (Rakibs511) [inline packages/ui/components/elements/GenericScrollbar/GenericScrollbar.styled.tsx:18]: we don't use px. please do it with rem

File: packages/ui/components/elements/GenericScrollbar/GenericScrollbar.styled.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    background-color: #f5f5f5;
+  }
+
+  /* Scrollbar */
+  &::-webkit-scrollbar {
+    width: 12px;
```
Comment (Rakibs511) [inline packages/ui/components/elements/GenericScrollbar/GenericScrollbar.styled.tsx:24]: px to rem

File: packages/ui/components/elements/GenericScrollbar/GenericScrollbar.styled.tsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  /* Thumb */
+  &::-webkit-scrollbar-thumb {
+    border-radius: 10px;
+    -webkit-box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.3);
+    background: linear-gradient(180deg, #d0368a, #708ad4);
+  }
+`
+
+export const ForceOverflow = styled.div`
+  min-height: 450px;
```
Comment (Rakibs511) [inline packages/ui/components/elements/GenericScrollbar/GenericScrollbar.styled.tsx:37]: px to rem

File: packages/ui/components/elements/GenericScrollbar/GenericScrollbar.styled.tsx [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+`
+
+export const ScrollbarStyles = styled.div`
+  /* Track */
+  &::-webkit-scrollbar-track {
+    -webkit-box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.3);
+    border-radius: 10px;
```
Comment (Rakibs511) [inline packages/ui/components/elements/GenericScrollbar/GenericScrollbar.styled.tsx:44]: px to rem

File: packages/ui/components/elements/GenericScrollbar/GenericScrollbar.test.tsx [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+      </GenericScrollbar>
+    );
+
+    const scrollbar = getByTestId('custom-scrollbar');
+    const content = getByText('Scrollable Content');
+
+    expect(scrollbar).toBeInTheDocument();
+    expect(content).toBeInTheDocument();
+
+    expect(scrollbar).toHaveStyle('height: 400px');
```
Comment (Rakibs511) [inline packages/ui/components/elements/GenericScrollbar/GenericScrollbar.test.tsx:19]: px to rem

**Comment by TanvirAlam** [comment]
Closing this as it is unnecessary for using this now, let wast time big fish

**Pull request closed without merging**